code/modules/projectiles/guns/sentries.dm

```diff
@@ -62,9 +62,10 @@
 	desc = "A deployable, semi-automated turret with AI targeting capabilities. Armed with a M30 autocannon and a 25-round box magazine. Use Alt-Click to remove its battery."
 	icon_state = "sentry"
 
-	turret_range = 8
 	deploy_time = 8 SECONDS
 	max_shells = 50
+	fire_delay = 0.2 SECONDS
```

Comment:
You upped the dps by 1.8 overall, not counting the pene, the sunder, the damage falloff, and the higer projectiles count

This is way too much, sentries are not that bad.

Reply:
yes they were, at some tiles you'd take 8 seconds to kill a single drone standing still.